### ISSUE TITLE
Not possible to add an empty field in the schema

### ISSUE THREAD

**Issue by michaelryancaputo** [bozzltron/node-json-transform#2]
When I add an empty field to the schema map, the first object of the input completes fine, however after that my operate functions nolonger work. I got around this by making a function to run in operate that returns an empty string.

the package should check to see if the value content is empty if it is return an empty string. Further, you should be able to define the type of the empty value - it may also be valuable to be able to define types for every value including empty ones.

**Comment by bozzltron** [comment]
Thanks for the feedback!  I need some clarity on your request.  Does "empty field to the schema map" mean the objects you are processing have empty values?  Are the "operate" mappings the only thing that is failing?

**Comment by michaelryancaputo** [comment]
Sorry, i shoudl have been more specific. My schema looks similar to this:

```
  'allowedEbcs': 'allowedEbcs',
  'promoImage': '',
  'squareImage': '',
  'stillImage': '',
  'tallImage': ''
```

An ideal solution would be to be able to set a type for those values.

An additional feature that I think would really help, would be to somehow add a type to all of the values, since my json contains a mix of Numbers, Strings, dates, etc i'm having to create operate functions to format those values.

**Comment by bozzltron** [comment]
I've released version 1.0.3 to handle this.

**Comment by michaelryancaputo** [comment]
Awesome! Thanks!
